packages/kit/src/core/start/index.js

```diff
@@ -48,7 +48,7 @@ export async function start({ port, host, config, https: use_https = false, cwd
 		read: (file) => fs.readFileSync(join(config.kit.files.assets, file))
 	});
 
-	const server = await get_server(use_https, config.kit, (req, res) => {
+	const server = await get_server(use_https, config.kit.vite, (req, res) => {
```

Comment:
the parameter name in `get_server` and in `dev` where it's being called (since https://github.com/sveltejs/kit/pull/1517) is `user_config`. It sounds like perhaps it should be renamed to `vite_config`. also the type is `any` which is allowing it to be different between the different places. it'd be nice if that could be updated to avoid a regression here

Reply:
I agree with the renaming and type usage.  There is a problem with Vite's type definitions for their config, but there will be a correction in the next patch release. Also, a lot of changes already have been made to dev/index.js in #1572.

I think we need to hold off at least until the next Vite patch release and the next SvelteKit patch release.  

I'm leaving for the US on 6/18, returning on 7/14, and should have a computer available.

Reply:
If you want to just do the renaming now and then merge this that'd be fine and we can add the Vite types later